CMakeLists.txt

```diff
@@ -357,8 +357,8 @@ if( BUILD_RUBY )
     # brew install autoconf automake libtool
     # brew link autoconf automake libtool
     ExternalProject_Add(Ruby
-      URL https://codeload.github.com/ruby/ruby/tar.gz/v2_5_1
-      URL_MD5 98b550a05fec0f7a43a22d6c37404fd4
+      URL https://codeload.github.com/ruby/ruby/tar.gz/v2_5_5
```

Comment:
Ubuntu 18.04 is on [Ruby 2.5.1](https://packages.ubuntu.com/bionic/ruby/ruby2.5) will this version be compatible?  Is it possible to patch 2.5.1 with the specific issue code?  Would that be safer or not?

Reply:
SketchUp 2019 is also on Ruby 2.5.1.  The CLI should have no issues, I am only concerned about the shared library openstudio.so when loaded by Ruby 2.5.1 (either system or in SketchUp).  Should be easy to test:

```
ruby -I/path/to/openstudio.so/dir/ -e "require 'openstudio'" -e "puts OpenStudio::Model::Model.new"
```

Reply:
I didn't consider the system version of ruby here on Unix, I updated the version on both sides so they would be the same, but the fixes we need are Windows specific. IMO it's best to just roll back the Unix ones in this case to minimize issues.

Reply:
For all of these question though, Ruby considers the API version to be 2.5.0, as we've seen in how folders are named when Ruby is installed, etc. So, as much as I trust Ruby I trust that the ABI/API is compatible and we'll have no problems with 2.5.x line.

Reply:
I reverted Unix back to 2.5.1 so there's no question here about Ubuntu.

Reply:
> 
> 
> SketchUp 2019 is also on Ruby 2.5.1. The CLI should have no issues, I am only concerned about the shared library openstudio.so when loaded by Ruby 2.5.1 (either system or in SketchUp). Should be easy to test:
> 
> ```
> ruby -I/path/to/openstudio.so/dir/ -e "require 'openstudio'" -e "puts OpenStudio::Model::Model.new"
> ```

Verified as loading and executing with a system installed 2.5.1 on Windows with my 2.5.5 compiled `.so`.